### ISSUE TITLE
🐋 Docker image looking for a missing Handbrake release

### ISSUE THREAD

**Issue by pilotdude** [automatic-ripping-machine/automatic-ripping-machine#797]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

When installing the latest docker image I'm running into an error where the script is looking for a handbrake release that is unavailable. 

Here is the exact error that I'm getting: 
![image](https://user-images.githubusercontent.com/11094614/226108752-0064ee5a-857c-4a1c-aca0-f96d68eb9ede.png)

I have checked and indeed it looks like that repository mentioned is empty :(. I dug around a bit in the docker.sh file trying to figure out where this repository was mentioned, I was thinking maybe I could manually change the mentioned release to one that does exist but I couldn't find where the release was mentioned. 

One thing to note is that I do have handbrake already installed, so it would be nice if I could just bypass this step anyway but I also couldn't figure out how to do that.

After trying these options I figured if I was having issues then probably there were others who were also having issues with the installer so I'm logging a bug. 

### To Reproduce

Run the docker image installer as per the wiki. 

I have handbrake already installed but I don't think that this is relevant to reproducing the error. 

### Built from source files ?

- [ ] I Built the image from source

### What architecture are you seeing the problem on?

Other (Please state in `Anything else` section)

### Expected behavior?

The installer to successfully install the version of handbrake that I need to run ARM. 

### Anything else?

I don't know what architecture I'm on :( 

If it is actually relevant let me know and I will figure it out! 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by shitwolfymakes** [comment]
What operating system are you using?

**Comment by shitwolfymakes** [comment]
Are you using the docker setup script or the ubuntu install script?

**Comment by pilotdude** [comment]
I'm using the docker setup script.

-Charlie

**Comment by shitwolfymakes** [comment]
That's bizarre. The docker setup script doesn't install handbrake. Did you upgrade the OS from 20.04?

**Comment by pilotdude** [comment]
No this machine is a clean install of Ubuntu, I had done very little with it before I tried to run the docker script. One of the only things that I had done was installing handbrake. 

I was intending to setup this machine to run as an ARM machine to make a digital copy of my library of DVDs/BlueRays as I had an issue when moving my collection onto an external storage device and I lost the entire collection.

Long story short, no I didn't upgrade the OS.

**Comment by 1337-server** [comment]
Closing as stale. If this is still an issue, please let us know. But going over the latest version of the installation script, there should be no installing of HandBrake.

Try updating to the latest install script for docker and try again.
